### PR TITLE
data: sheriffOladimeji

### PR DESCRIPTION
**Pull request by sheriffOladimeji**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4640"></a>

### CODE REVIEW

File: data/sheriffOladimeji.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Sheriff Oladimeji",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend web developer, CS student"
```
Comment (priyankarpal) [inline data/sheriffOladimeji.json:5]: hey, @sheriffOladimeji just add a comma.
``` suggestion
  "bio": "Frontend web developer, CS student",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — DISMISSED** [review]
Looks good

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/sheriffOladimeji.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Sheriff Oladimeji",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/sheriffOladimeji.json:3]: We no longer need the `displayStatsPublic` key. So please remove this field from your json.

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center"></p>

**Pull request merged**